src/store/getters/state-getters.ts

```diff
@@ -411,3 +425,34 @@ export const getFileNumber = (state: StateIF): string => {
 export const getHasPlanOfArrangement = (state: StateIF): boolean => {
   return state.stateModel.newAlteration.courtOrder.hasPlanOfArrangement
 }
+
+/** Get boolean indicating if the share structure contains any special rights of restrictions. */
+export const getHasRightsOrRestrictions = (state: StateIF): any => {
+  const shareClasses = state.stateModel.shareStructureStep.shareClasses
+  const hasRightsOrRestrictionsArr = []
+
+  // Isolate the hasRightsOrRestrictions boolean values
+  shareClasses.forEach(shareClass => {
```

Comment:
Could you use `some` instead? Theoretically faster (and slicker ;) ).

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
I'll check it out.. no promises though 😁 lol

Reply:
In the old days, we'd `break` out of "for" loops to stop searching when we found a match. Am I (carbon) dating myself, lol?

Reply:
hahah you've been exposed! jk jk

Reply:
Updated!